bundle-size/action.yml

```diff
@@ -85,13 +89,20 @@ runs:
 
     - name: Generate stats report and comment on PR
       if: ${{ github.event_name == 'pull_request' && steps.main-stats-info.outputs.run-id != '' }}
-      uses: actions/github-script@v7
+      uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
       with:
         script: |
           const script = require('${{ github.action_path }}/index.js')
-          await script({github, context, core}, ${{ inputs.threshold }}, "${{ inputs.working-directory }}/stats.json", "${{ inputs.working-directory }}/pr-stats.json", "${{ inputs.working-directory }}")
```

Comment:
nit: since these are arguments to a javascript function, IMO you can just zizmor-ignore it but the change also looks good.